pkg/schema/testHelper_test.go

```diff
@@ -0,0 +1,100 @@
+// Copyright Contributors to the Open Cluster Management project
+package schema
```

Comment:
Let's try to avoid duplicating this file.
- Could we reference these from the resolver package?
- Could we combine the schema and resolver packages into a single package?

Reply:
There are a few differences between the two. is it possible to overload the Scan Method and Row structs?